cmd/rtrmon/rtrmon.go

```diff
@@ -55,6 +55,7 @@ var (
 
 	UserAgent                  = flag.String("useragent", fmt.Sprintf("StayRTR-%v (+https://github.com/bgp/stayrtr)", AppVersion), "User-Agent header")
 	DisableConditionalRequests = flag.Bool("disable.conditional.requests", false, "Disable conditional requests (using If-None-Match/If-Modified-Since headers)")
+	GracePeriod                = flag.Duration("grace.period", time.Minute*20, "Grace period during which objects removed from a source are not considered for the diff")
```

Comment:
I'm just skimming really and no expert in Golang either.
Is there a way to set this GracePeriod via configuration file?